conf/experiments/lightgbm-inferencing.yaml

```diff
@@ -35,6 +35,7 @@ lightgbm_inferencing_config:
       build: docker/lightgbm-v3.2.1/linux_cpu_mpi_build.dockerfile
     - framework: lightgbm_c_api # v3.2.1 with C API prediction
       build: docker/lightgbm-custom/v321_patch_cpu_mpi_build.dockerfile
```

Comment:
Do we need a different build for lightgbm_ray?
Where is src/scripts/inferencing/lightgbm_ray/default.dockerfile be used?

Reply:
the lightgbm_ray module contains its own default.dockerfile here https://github.com/microsoft/lightgbm-benchmark/blob/jfomhover/rayscripthelper/src/scripts/inferencing/lightgbm_ray/default.dockerfile

in this config file lightgbm-inferencing.yaml you can override this dockerfile with your own using this `build:` setting. but in the case of this lightgbm_ray component, we don't need to for now.